Tests/TuistKitTests/ProjectEditor/ProjectEditorMapperTests.swift

```diff
@@ -103,7 +133,7 @@ final class ProjectEditorMapperTests: TuistUnitTestCase {
         XCTAssertEqual(runAction.arguments, Arguments(launchArguments: [generateArgument: true]))
     }
 
-    func test_edit_when_there_are_no_helpers() throws {
+    func test_edit_when_there_are_no_helpers_are_no_setup_are_no_config() throws {
```

Comment:
nitpick: `and`s instead of `are`s, optionally you can drop the conjunction completely